packages/console/src/pages/Applications/index.tsx

```diff
@@ -69,7 +71,13 @@ const Applications = () => {
             </tr>
           )}
           {data?.map(({ id, name, type }) => (
-            <tr key={id}>
+            <tr
+              key={id}
+              className={styles.clickable}
+              onClick={() => {
```

Comment:
nit: would simply `onClick  = {() => navigate()}` work? or must be void.

Reply:
eslint auto fix this based on return type - in this case it's void